www/src/pages/table/NodeSearchComponent.tsx

```diff
@@ -44,62 +55,121 @@ export const NodeSearchComponent = withStyles(styles)(
       onDeselect,
       onSelect,
       classes,
+      classSearchOnly,
     }: {
       sm: SemanticModel;
       value?: SearchValue;
       onDeselect: (value: SearchValue) => void;
       onSelect: (value: SearchValue) => void;
+      classSearchOnly: boolean;
     } & WithStyles<typeof styles>) => {
       const { classStore } = useStores();
-      const seqInvoker = new SequentialFuncInvoker();
+      const [searchOptions, setSearchOptions] = useState<SearchOptions[]>();
 
       // gather all options already in the store, leverage the fact
       // that property store is readonly
       const options = useMemo(() => {
-        const options: ({
-          value: string;
-          label: string;
-        } & SearchValue)[] = [];
-        for (const r of classStore.iter()) {
-          options.push({
-            type: "class",
-            id: r.id,
-            value: `class:${r.id}`,
-            label: sm.graph.uriCount.nextLabel(r.uri, r.readableLabel),
-          });
-        }
+        const options: SearchOptions[] = [];
 
         for (const u of sm.graph.nodes) {
+          if (
+            classSearchOnly &&
```

Comment:
rename the property `classSearchOnly` to match its purpose

Reply:
fixed